### PR TITLE
DEP-3057 : Fix End chat dialog for screen readers

### PR DESCRIPTION
**Pull request by lmugnai**
# DEP-(StoryNumberHere)

## Checklist PR Raiser
 - [x]  I've ensured code coverage has not decreased
 - [x]  I've dealt with any new compilation warnings
 - [x]  I've ensured the team's coding standards have been met (TBC)?
 - [ ]  If relevant, I've created corresponding app-config-XXX changes?
 - [x]  If I've created new test data, I've ensured there is no real person's data (see https://confluence.tools.tax.service.gov.uk/display/CD/Test+Data+in+the+Open) 
 - [x]  I've considered the impact my changes have on the UI/Journey tests (please do not break them)
-  [ ]  If relevant, I've added links to associated PRs

## Checklist PR Reviewer
 - [x]  I've checked to ensure all relevant unit tests have been written
 - [x]  I've checked to ensure the team's coding standards have been met
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  If I merge I will ensure I use squash and merge

### CODE REVIEW

File: test/uk/gov/hmrc/digitalengagementplatformskin/javascripts/views/EndChatPopup.spec.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	test('Escape key closes end chat dialogie', () => {
+        jest.spyOn(ecp, 'onCancelEndChat').mockImplementation();
+
+        var event = new KeyboardEvent('keydown', {'key': 'Escape'});
+        wrapper.dispatchEvent(event);
+
+        expect(ecp.onCancelEndChat).toHaveBeenCalled();
+	});
+
+
```
Comment (tonymyers) [inline test/uk/gov/hmrc/digitalengagementplatformskin/javascripts/views/EndChatPopup.spec.js:31]: Remove extra blank line

File: test/uk/gov/hmrc/digitalengagementplatformskin/javascripts/views/EndChatPopup.spec.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    test('Click on confirm end chat and end chat', () => {
+        jest.spyOn(ecp, 'onConfirmEndChat').mockImplementation();
+
+        var event = new KeyboardEvent('click', {});
+        wrapper.querySelector("#confirmEndChat").dispatchEvent(event);
+
+        expect(ecp.onConfirmEndChat).toHaveBeenCalled();
+	});
+});
```
Comment (tonymyers) [inline test/uk/gov/hmrc/digitalengagementplatformskin/javascripts/views/EndChatPopup.spec.js:49]: Add a blank line after last line of code

**Review by tonymyers — APPROVED** [review]

File: test/uk/gov/hmrc/digitalengagementplatformskin/javascripts/views/EndChatPopup.spec.js (hunk 1 above)
Comment (lmugnai) [reply test/uk/gov/hmrc/digitalengagementplatformskin/javascripts/views/EndChatPopup.spec.js]: done

File: test/uk/gov/hmrc/digitalengagementplatformskin/javascripts/views/EndChatPopup.spec.js (hunk 2 above)
Comment (lmugnai) [reply test/uk/gov/hmrc/digitalengagementplatformskin/javascripts/views/EndChatPopup.spec.js]: done

**Pull request merged**